packages/alfa-style/src/property/contain.ts

```diff
@@ -0,0 +1,120 @@
+import {
+  Keyword,
+  Token,
+  Tuple,
+  type Parser as CSSParser,
+} from "@siteimprove/alfa-css";
+import { Parser } from "@siteimprove/alfa-parser";
+import { Slice } from "@siteimprove/alfa-slice";
+import { Err, Result } from "@siteimprove/alfa-result";
+
+import { Longhand } from "../longhand.js";
+
+const { either, map } = Parser;
+
+type Specified = Specified.Single | Specified.Multiple;
+type Computed = Specified;
+
+namespace Specified {
+  export type Single = Tuple<
+    [Keyword<"none"> | Keyword<"strict"> | Keyword<"content">]
+  >;
```

Comment:
**Suggestion:** Does that need to be wrapped in a `Tuple`? Can't it be `Keyword<"none"> | Keyword<"strict"> | Keyword<"content">`?